editor.planx.uk/src/@planx/components/List/schemas/Adverts.ts

```diff
@@ -0,0 +1,40 @@
+import { Schema } from "@planx/components/List/model";
+
+export const ProposedAdvertisements: Schema = {
+  type: "Proposed advertisements",
+  fields: [
+    {
+      type: "number",
+      data: {
+        title: "How many fascia signs are you applying for?",
+        fn: "fascia",
+        allowNegatives: false,
+      },
+    },
+    {
+      type: "number",
+      data: {
+        title: "How many projecting or hanging signs are you applying for?",
+        fn: "projecting",
+        allowNegatives: false,
+      },
+    },
+    {
+      type: "number",
+      data: {
+        title: "How many hoardings are you applying for?",
+        fn: "hoarding",
+        allowNegatives: false,
+      },
+    },
+    {
+      type: "number",
+      data: {
+        title: "How many other advertisements are you applying for?",
+        fn: "other",
+        allowNegatives: false,
+      },
+    },
+  ],
+  min: 0,
```

Comment:
nit: when `min: 0`, the component actually confusing loads like this - so we probably really want `min: 1` ?

![Screenshot from 2024-06-10 20-17-04](https://github.com/theopensystemslab/planx-new/assets/5132349/14b56fee-0884-4274-8523-a42a3cd5cb76)

if we want it to be possible to "continue" without submitting any items, maybe there's a better way we can control that in the Editor modal using a simple toggle rather than in the schema via `min` ?

Reply:
Thanks, I misunderstood what was being set by min and max. Looks like min: 0 isn't even possible, as it throws this when you try to add any item: 
![image](https://github.com/theopensystemslab/planx-new/assets/118665588/af40623c-62e2-4ae7-bf24-a478c19f8899)